core/src/main/java/greencity/controller/EcoNewsController.java

```diff
@@ -113,4 +117,28 @@ public EcoNewsController(EcoNewsService ecoNewsService) {
         ecoNewsService.delete(econewsId);
         return ResponseEntity.status(HttpStatus.OK).build();
     }
+
+    /**
+     * Method for getting all eco news by tags and language.
+     *
+     * @param getEcoNewsDto - - dto for search {@link EcoNewsDto} by tags and language
+     * @return list of {@link EcoNewsDto} instances.
+     * @author Kovaliv Taras.
+     */
+    @ApiOperation(value = "Get eco news by tags")
+    @ApiResponses(value = {
+        @ApiResponse(code = 200, message = HttpStatuses.OK),
+        @ApiResponse(code = 400, message = HttpStatuses.BAD_REQUEST),
+        @ApiResponse(code = 403, message = HttpStatuses.FORBIDDEN),
+    })
+    @PostMapping("/tags")
+    @ApiPageable
+    public ResponseEntity<PageableDto<EcoNewsDto>> getEcoNews(@ApiIgnore Pageable page,
+                                                              @RequestBody GetEcoNewsDto getEcoNewsDto) {
```

Comment:
If GetEcoNewsDto is an object with search properties better naming would be SearchCriteriaEcoNewsDto